### PR TITLE
1792 higlight metadata

### PR DESCRIPTION
**Pull request by habbes**
Fixes issue #1792

PR checklist:
- [ ] Update READ.me ?
- [ ] Update API documentation ?

QA checklist:
- [ ] Smoke test the functionality described in the issue
- [ ] Test for side effects
- [ ] UI responsiveness
- [ ] Cross browser testing
- [ ] Code review

### CODE REVIEW

File: app/api/search/search.js [hunk 1]
```diff
@@ -56,6 +56,37 @@ function agregationProperties(properties) {
...
 }
 
+function snippetsFromSearchHit(hit) {
+  let snippets = [];
+  if (hit.inner_hits && hit.inner_hits.fullText.hits.hits.length) {
+    const regex = /\[\[(\d+)\]\]/g;
+
+    const fullTextHighlights = hit.inner_hits.fullText.hits.hits[0].highlight;
+
+    snippets = fullTextHighlights[Object.keys(fullTextHighlights)[0]].map((snippet) => {
```
Comment (konzz) [inline app/api/search/search.js]: I suggest extracting this `Object.keys(fullTextHighlights)[0]` to a variable:
`const fullTextLanguageKey = Object.keys(fullTextHighlights)[0]`

Comment (habbes) [reply app/api/search/search.js]: Alright

**Comment by txau** [comment]
Add the label of the field the snippet is coming from in the card. So in the left card, the snippet is coming from the document, we could add a "Document contents" label (make it translatable).

The right card is coming from the field "Resumen".

screenshot from 2018-08-01 14-55-33

- [ ] Metadata results will be displayed at the beginning of the list.
- [ ] We will use the same styles as the labels in the documents.
- [ ] All the snippets for a particular metadata field will be displayed under their label without page numbers or any other label.
- [ ] The label will be clickable and will lead to the entity/document metadata page.

screenshot from 2018-08-01 15-02-27
screenshot from 2018-08-01 15-02-18

**Comment by txau** [comment]
Related: https://github.com/huridocs/uwazi/issues/1391

**Comment by txau** [comment]
@habbes I just noticed when I run yarn install on this branch there are some uncommitted changes:
	modified:   yarn.lock

**Pull request merged**